server/matchmaker/matchmaker_queue.py

```diff
@@ -278,6 +278,10 @@ def to_dict(self):
         """
         Return a fuzzy representation of the searches currently in the queue
         """
+        if self.team_size == 1:
+            ratings = {round(search.ratings[0].mean / 25) * 25 for search in self._queue.keys()}
```

Comment:
Would it make sense to add a config value for the granularity?

Reply:
I don't see a reason for this. We should choose a reasonable value once and then there is no real reason to change it on the fly. I will extract the number into a variable though to make it less of a magical number.